dotnet/Directory.Packages.props

```diff
@@ -9,13 +9,15 @@
     <PackageVersion Include="Azure.Identity" Version="1.10.3" />
     <PackageVersion Include="Azure.Monitor.OpenTelemetry.Exporter" Version="1.0.0" />
     <PackageVersion Include="Azure.Search.Documents" Version="11.5.0-beta.5" />
+    <PackageVersion Include="Handlebars.Net.Helpers" Version="2.4.1" />
     <PackageVersion Include="Markdig" Version="0.33.0" />
     <PackageVersion Include="Handlebars.Net" Version="2.1.4" />
     <PackageVersion Include="JsonSchema.Net.Generation" Version="3.5.0" />
     <PackageVersion Include="Microsoft.Azure.Kusto.Data" Version="11.3.5" />
     <PackageVersion Include="Microsoft.Bcl.HashCode" Version="[1.1.0, )" />
     <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
     <PackageVersion Include="Microsoft.Bcl.TimeProvider" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.SemanticKernel.PromptTemplate.Handlebars" Version="1.0.0-beta8" />
```

Comment:
Wait...  What?  Ingesting this code through a NuGet instead of as a project reference?  Is this what we want to do?

Reply:
somehow this was auto-added, removing